problems/2.functions.js

```diff
@@ -2,34 +2,44 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
+  let name = null
+  return name
```

Comment:
Needs to console.log . Don't set name to null. Name is given to you by the person invoking your function.